src/folderView.js

```diff
@@ -1,4 +1,5 @@
 import { getClassNameForMimeType, getClassNameForFilename } from 'font-awesome-filetypes'
+const emojiRegex = require('emoji-regex/RGI_Emoji.js');
```

Comment:
Can this be imported with something like below?

```javascript
import emojiRegex from 'emoji-regex/RGI_Emoji.js'
```

Also no semicolon at the end plz :D

Reply:
Fixed